modules/gateway/controllers/service/populate/virtualservice.go

```diff
@@ -264,7 +264,7 @@ func isProtocolSupported(protocol v1.Protocol) bool {
 }
 
 func VirtualServiceFromSpec(aggregated bool, systemNamespace string, name, namespace string, clusterDomain *projectv1.ClusterDomain, service *v1.Service, dests ...Dest) *v1alpha3.VirtualService {
```

Comment:
Could we add a comment to define what `aggregated` does?

Reply:
My understanding is that aggregated determines if its the route to a revision versus the "app" itself.